instrumentation/jaxws/jaxws-2.0-cxf-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cxf/TracingEndInInterceptor.java

```diff
@@ -5,13 +5,15 @@
 
 package io.opentelemetry.javaagent.instrumentation.cxf;
 
+import org.apache.cxf.interceptor.OutgoingChainInterceptor;
 import org.apache.cxf.message.Message;
 import org.apache.cxf.phase.AbstractPhaseInterceptor;
 import org.apache.cxf.phase.Phase;
 
 public class TracingEndInInterceptor extends AbstractPhaseInterceptor<Message> {
   public TracingEndInInterceptor() {
     super(Phase.POST_INVOKE);
+    addBefore(OutgoingChainInterceptor.class.getName());
```

Comment:
```suggestion
    // end the span before the OutInterceptors (callbacks) are executed
    addBefore(OutgoingChainInterceptor.class.getName());
```

Reply:
Thanks for the review.
Sorry for the missing test, I just added it.